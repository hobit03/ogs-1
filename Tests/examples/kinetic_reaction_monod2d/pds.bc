GeoSys-BC: Boundary Conditions ------------------------------------------------
#BOUNDARY_CONDITION
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT 1.01

 
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_A
 $GEO_TYPE
  POLYLINE PLY_S1
 $DIS_TYPE
  CONSTANT 0.0

#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_A
 $GEO_TYPE
  POLYLINE PLY_S2
 $DIS_TYPE
  CONSTANT 3.3e-4
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_A
 $GEO_TYPE
  POLYLINE PLY_S3
 $DIS_TYPE
  CONSTANT 0.0
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_B
 $GEO_TYPE
  POLYLINE PLY_S1
 $DIS_TYPE
  CONSTANT 2.5e-4

#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_B
 $GEO_TYPE
  POLYLINE PLY_S2
 $DIS_TYPE
  CONSTANT 0.0
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_B
 $GEO_TYPE
  POLYLINE PLY_S3
 $DIS_TYPE
  CONSTANT 2.5e-4

#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  Comp_C
 $GEO_TYPE
  POLYLINE PLY_BC_LEFT
 $DIS_TYPE
  CONSTANT 0.0   
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  ConsTracer
 $GEO_TYPE
  POLYLINE PLY_S1
 $DIS_TYPE
  CONSTANT 0.0 
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  ConsTracer
 $GEO_TYPE
  POLYLINE PLY_S2
 $DIS_TYPE
  CONSTANT 1
  
#BOUNDARY_CONDITION
 $PCS_TYPE
  KIN_REACT_GIA
 $PRIMARY_VARIABLE 
  ConsTracer
 $GEO_TYPE
  POLYLINE PLY_S3
 $DIS_TYPE
  CONSTANT 0.0 
  
#STOP

