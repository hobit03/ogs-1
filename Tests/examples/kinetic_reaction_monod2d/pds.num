GeoSys-NUM: Numerical Parameter ----------------------------------------
#NUMERICS
 $PCS_TYPE
  GROUNDWATER_FLOW
 $ELE_GAUSS_POINTS
  3
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  2      6 1.e-014       10000           1.0   1       2
#NUMERICS
 $PCS_TYPE
  MASS_TRANSPORT 
 $ELE_GAUSS_POINTS
  3
 $LINEAR_SOLVER 
; method error_tolerance max_iterations theta precond storage
  2      6 1.e-014       1000           1.0   1       2 
#STOP
