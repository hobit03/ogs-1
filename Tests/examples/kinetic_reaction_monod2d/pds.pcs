GeoSys-PCS: Processes ------------------------------------------------
#PROCESS
 $PCS_TYPE
 GROUNDWATER_FLOW
 $NUM_TYPE
   NEW  
#PROCESS
 $PCS_TYPE
  KIN_REACT_GIA
 $NUM_TYPE
   NEW  
#STOP



#PROCESS
 $PCS_TYPE
 GROUNDWATER_FLOW
 $NUM_TYPE
   NEW  
 $TIM_TYPE
   STEADY
 $ST_RHS
   1
 $BOUNDARY_CONDITION_OUTPUT    

#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW  
 $MEMORY_TYPE
  1
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW  
 $MEMORY_TYPE
  1
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW  
 $MEMORY_TYPE
  1
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW  
 $MEMORY_TYPE
  1
#PROCESS
 $PCS_TYPE
  MASS_TRANSPORT
 $NUM_TYPE
   NEW  



