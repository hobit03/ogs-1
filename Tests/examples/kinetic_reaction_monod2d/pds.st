GeoSys-ST: Source Terms ------------------------------------------------
#SOURCE_TERM
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE PLY_OUT1
 $DIS_TYPE
  CONSTANT_GEO  -8.75112917795845E-07 ; q=0.5m/d
#SOURCE_TERM
 $PCS_TYPE
  GROUNDWATER_FLOW
 $PRIMARY_VARIABLE 
  HEAD
 $GEO_TYPE
  POLYLINE PLY_OUT2
 $DIS_TYPE
  CONSTANT_GEO -2.82294489611563E-07 ; q=0.5m/d

#STOP

