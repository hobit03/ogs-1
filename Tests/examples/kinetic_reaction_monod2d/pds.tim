GeoSys-TIM: Timestepping ------------------------------------------------
#TIME_STEPPING
 $PCS_TYPE
  GROUNDWATER_FLOW
 $TIME_STEPS
   100000  216.0
 $TIME_END
  43200000.0
 $TIME_START
  0.0
#TIME_STEPPING
 $PCS_TYPE
  MASS_TRANSPORT
 $TIME_STEPS
   100000  216.0
 $TIME_END
  43200000.0
 $TIME_START
  0.0
#STOP

