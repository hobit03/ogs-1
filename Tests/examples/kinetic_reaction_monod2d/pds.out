GeoSys-OUT: Output ------------------------------------------------
#OUTPUT // profile
 $NOD_VALUES
  HEAD
  ConsTracer
  Comp_A
  Comp_B
  Comp_C
  Biomass
 $GEO_TYPE
  DOMAIN
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 20000   

#OUTPUT // profile
 $NOD_VALUES
  HEAD
  ConsTracer
  Comp_A
  Comp_B
  Comp_C
  Biomass
 $GEO_TYPE
  POLYLINE PLY_4
 $DAT_TYPE
  TECPLOT
 $TIM_TYPE
  STEPS 20000  
#STOP

